docs/releasing/make-docker-images.sh

```diff
@@ -42,14 +42,6 @@ while IFS= read -d $'\0' -r dir; do
 
         log "Pushing: ${image}"
         docker push "${image}"
-
-        if [ $svcname != "frontend" ] && [ $svcname != "loadgenerator" ]
-        then
-            log "Building: ${image}-native-grpc-probes"
-            docker build --pull -t "${image}-native-grpc-probes" . --target without-grpc-health-probe-bin
-            log "Pushing: ${image}-native-grpc-probes"
-            docker push "${image}-native-grpc-probes"
-        fi
```

Comment:
* I'm removing the code that builds and pushes `-native-grpc-probes` Docker images for each gRPC microservice.
* I have not tested my change to this `make-docker-images.sh` script, but I will find out soon during the next release. This should be fine.
* frontend and loadgenerator are not gRPC service so, this doesn't apply to them.